src/reporter/query_1T1E1A.py

```diff
@@ -42,13 +47,20 @@ def query_1T1E1A(attr_name,   # In Path
                                    limit=limit,
                                    offset=offset,
                                    fiware_service=fiware_s,
-                                   fiware_servicepath=fiware_sp,)
+                                   fiware_servicepath=fiware_sp,
+                                   geo_query=geo_query)
     except AmbiguousNGSIIdError as e:
         return {
             "error": "AmbiguousNGSIIdError",
             "description": str(e)
         }, 409
 
+    except ValueError:
```

Comment:
I'd suggest putting the new code in its own try-catch. If something inside query triggers `ValueError` at least it won't return a misleading message.

Not urgent, but if possible, it'd be nice to give a hint of what was wrong in the parameters.